presenter/GraphSynthesizer.py

```diff
@@ -0,0 +1,8 @@
+def get_book_identifier(row) -> str:
+    if "book_name" in row:
+        return row["book_name"]
+    return row["book_id"]
+
+
+def compute_presented_name(book_dataframe):
+    return book_dataframe.apply(get_book_identifier, axis=1)
```

Comment:
If your function is only one line, and is used only in a single place, maybe it shouldn't be a function..

Reply:
I plan on inlining get_book_identifier, other than that, its parent is already too big (you've already c ommented about it).
I think that after refactoring parent, it will be clearer wether or not this function should be extracted or not.